package-lock.json

```diff
@@ -1,8 +1,11038 @@
 {
     "name": "pmi-drc-hpo",
     "version": "1.0.0",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
May need to ensure we're all up to npm v7.x for this to not flip back and forth.